modules/programs/exa.nix

```
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.exa;
+
+  aliases = {
+    ls = "${pkgs.exa}/bin/exa";
+    ll = "${pkgs.exa}/bin/exa -l";
+    la = "${pkgs.exa}/bin/exa -a";
+    lt = "${pkgs.exa}/bin/exa --tree";
+    lla = "${pkgs.exa}/bin/exa -la";
+  };
+
+in {
+  meta.maintainers = [ maintainers.kalhauge ];
+
+  # Exa is modern replacement for 'ls'
+  # https://github.com/ogham/exa
```

This comment seems superfluous since it's already described by the nix package itself: https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/tools/misc/exa/default.nix#L38

(I'm not very familiar with the standards in home-manager, though, so take this recommendation with a grain of salt.)

---

You might be right! I do however like that a link to the documentation is easily available in some of the other modules.

---

Can put the "modern replacement" part in the `enable` description. Something like
```suggestion
    enable = mkEnableOption "exa, a modern replacement of <command>ls</command>";
```
perhaps.